packages/tokens/src/tokens.json

```diff
@@ -1731,6 +1731,191 @@
       "description": "Default shadow"
     }
   },
+  "_docs": {
+    "components": {
+      "button": {
+        "variants": {
+          "title": {
+            "value": "string",
+            "type": "text",
+            "description": "Variants"
+          },
+          "short": {
+            "value": "string",
+            "type": "text",
+            "description": "There are a number of button variations available. Be sure to select the right one for the action you want to make available."
```

Comment:
"description": "There are a number of button variants available. Be sure to select the right one for the action you want to make available."